<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KV58FirmwareDlg</class>
 <widget class="QDialog" name="KV58FirmwareDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KV58 Firmware Download</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/terminal16.png</normaloff>:/terminal16.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="vlMain">
   <item>
    <widget class="QFrame" name="frUpper">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>62</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="hlUpperTop">
        <item>
         <widget class="QLabel" name="lbFirmwarePath">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Path:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbPath">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbFirmwarePath">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hlUpperBottom">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QPushButton" name="pbDownload">
          <property name="text">
           <string>Download APP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbDownloadBootLoader">
          <property name="text">
           <string>Download LDR</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbGetApplication">
          <property name="text">
           <string>Get APP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbGetROM">
          <property name="text">
           <string>Get ROM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbGetLoader">
          <property name="text">
           <string>Get LDR</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="prDownload">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbClearCommList">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbStopScrollCommText">
          <property name="text">
           <string>Stop Scroll</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Console" name="lvCommText">
     <property name="font">
      <font>
       <family>JetBrains Mono</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frDown">
     <layout class="QHBoxLayout" name="hlDown">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pbReboot">
        <property name="text">
         <string>Boot Loader</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbBootApp">
        <property name="text">
         <string>Boot App</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hsDownLeft">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>131</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbVersion">
        <property name="text">
         <string>Version</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbSizes">
        <property name="text">
         <string>Size</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hsDownRight">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>131</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbClose">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Console</class>
   <extends>QPlainTextEdit</extends>
   <header>console.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
